### ISSUE TITLE
Pyside doesn't work with the newer verison of python

### ISSUE THREAD

**Issue by cyian-1756** [PornSieve/porn_sieve#2]
Pyside only works with python (2, 6), (2, 7), (3, 2), (3, 3), (3, 4), meaning those of us on python 3.5+ are left out. It'd be nice if you could either note that in your readme or even better (and far less likely) change up your gui toolkit

**Comment by PornSieve** [comment]
You're right, I ran into the same issue. I'll need to work on at least providing a TkInter implementation so this sort of thing can't happen again.

**Comment by awnumar** [comment]
I ran the install script for Pyside by first modifying it so that it didn't complain about the versions. Worked like a charm.

**Comment by Dan-Veinberg** [comment]
@awnumar  How did you install Pyside? I am using Python 3.8.2 and it will not let me install.

**Comment by awnumar** [comment]
> @awnumar  How did you install Pyside? I am using Python 3.8.2 and it will not let me install. 

I dunno man it was 4 years ago.

**Comment by Dan-Veinberg** [comment]
@awnumar  Can you try to remember it?
